### DISCUSSION TITLE
Retrieving translations in PageServerLoad methods

### DISCUSSION THREAD

**Discussion by martinbeentjes** [sveltekit-i18n/lib#119] [Q&A] [answered]
I want to internationalize my error message which might have dynamic variables inside them. In the `throw error()` construct I can get the translation using `t.get('key')`. However, I am unsure how to propagate the dynamic variable I want to have in the text.

**Comment by jarda-svoboda** [comment]
Could you, please, provide some repro?

**Comment by jarda-svoboda** [comment]
Anyway, [locale-router](https://github.com/sveltekit-i18n/lib/tree/master/examples/locale-router) or [locale-router-static](https://github.com/sveltekit-i18n/lib/tree/master/examples/locale-router-static) examples might help..

[a comment by an automated account is not shown]
